packages/plugin-react/tsdown.config.ts

```diff
@@ -0,0 +1,21 @@
+import { defineConfig } from 'tsdown'
+
+export default defineConfig({
+  entry: 'src/index.ts',
+  format: ['esm', 'cjs'],
```

Comment:
Should we go to esm only at the same time?

Reply:
I think we can do it in a separate PR.